README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
```

Comment:
There should be some more steps because I think Tetris also has controls to move the shape quicker down to the bottom of the screen. Also, what are exactly rows?